target/scripts/startup/setup.d/getmail.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+function _setup_getmail
+{
+  _log 'trace' 'Preparing Getmail configuration'
+
+  local GETMAILRC ID CONFIGS
+
+  GETMAILRC='/etc/getmailrc.d'
+  CONFIGS=false
```

Comment:
While I started out on this project using `false` and `true` as well, it has become the norm in DMS scripts to use `0` for `false` and `1` for `true`. I you want to know about why, you can ask me of course :)

```suggestion
  CONFIGS=0
```